rest/jersey2/src/main/java/org/seedstack/seed/rest/jersey2/internal/CacheControlFeature.java

```diff
@@ -63,9 +62,19 @@ public void filter(ContainerRequestContext requestContext,
             switch (this.policy) {
                 case NO_CACHE:
                     MultivaluedMap<String, Object> headers = responseContext.getHeaders();
-                    headers.putSingle(HttpHeaders.LAST_MODIFIED, new Date());
```

Comment:
Removing this header was intended?

Reply:
I think this is intentional because this header is only making sense when you want to actually cache the resource.